spec/requests/api/journalist_can_create_article_spec.rb

```diff
@@ -1,5 +1,14 @@
 RSpec.describe 'POST /articles', type: :request do
-  let(:headers) { { HTTP_ACCEPT: 'application/json' } }
+  let(:journalist) { create(:user, role: 'journalist') }
+  let(:journalist_credentials) { journalist.create_new_auth_token }
+  let(:journalist_headers) do
+    { HTTP_ACCEPT: 'application/json' }.merge!(journalist_credentials)
```

Comment:
what does require merge do?

Reply:
Similar logic about journalists having to be logged in to fill out the form.